### ISSUE TITLE
Disabling notifications generates warnings in log

### ISSUE THREAD

**Issue by bartlibert** [tiliado/nuvolaplayer#227]
**This issue has been fixed in the [master](https://github.com/tiliado/nuvolaplayer) & [3.0.x](https://github.com/tiliado/nuvolaplayer/tree/3.0.x) GIT branches. Thanks for your bug report.**

At first, this fix will be available with the next rebuilt of packages of the [Nuvola Player 3.1 Rolling Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.1/install.html). Later, the fix will be also included in the next bug-fix release of the [Nuvola Player 3.0 Stable Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.0/install.html).  **Update:** The fix has been released in Nuvola Player 3.0.3.

---

When I disable the "Notifications" component, I see a lot of warnings in the system log when a track is changed. There is no functional impact, but this makes debugging issues with Nuvola (and other tools) a bit more difficult because of the extra warnings.

Some example warnings (they are all very similar, but for different messages)
```
nuvolaplayer3-google-play-music.desktop[4243]: Worker: [CRITICAL Nuvola] WebExtension.vala:223: Failed to send message 'Nuvola.Notification.update'. No handler for message 'Nuvola.Notification.update'
nuvolaplayer3-google-play-music.desktop[4243]: Worker: [CRITICAL Nuvola] Application will not function properly.
```

When disabling the notifications component in the preferences, I see the following errors:
```
nuvolaplayer3-google-play-music.desktop[4243]: Runner: [CRITICAL DioriteGtk] diorite_actions_registry_remove_action: assertion 'action != NULL' failed
nuvolaplayer3-google-play-music.desktop[4243]: Runner: [CRITICAL DioriteGtk] Application will not function properly.
```

My system:
Debian stretch/sid
x86_64
Gnome 3.20

Nuvola Player R 3.1.0+201605031912.e5361ae
Revision e5361ae946d71b3593651dc89fa0d8b8e3ca84c2
WebKitGTK 2.12.2

Installed from tiliado sid devel debian repo

**Comment by fenryxo** [comment]
Hello @bartlibert. Thanks for your bug report.

**Comment by fenryxo** [comment]
**This issue has been fixed in the [master](https://github.com/tiliado/nuvolaplayer) & [3.0.x](https://github.com/tiliado/nuvolaplayer/tree/3.0.x) GIT branches. Thanks for your bug report.**

At first, this fix will be available with the next rebuilt of packages of the [Nuvola Player 3.1 Rolling Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.1/install.html). Later, the fix will be also included in the next bug-fix release of the [Nuvola Player 3.0 Stable Releases](https://tiliado.github.io/nuvolaplayer/documentation/3.0/install.html). To get informed when Nuvola Player 3.0.3 Stable Release is released,

  * follow Nuvola Player on [Facebook](https://www.facebook.com/nuvolaplayer),
    [Google+](https://plus.google.com/110794636546911932554) or
    [Twitter](https://twitter.com/NuvolaPlayer).
  * subscribe to the Nuvola Player Newsletter: [weekly](http://eepurl.com/bLbm5H)
    or [monthly](http://eepurl.com/bLbtM1).

**Comment by fenryxo** [comment]
Builds for Debian Sid will be available later as there are currently some build issues to be investigated. [Pending vala transition](https://release.debian.org/transitions/html/auto-vala.html) might be the cause.

**Comment by fenryxo** [comment]
The fix has been released in Nuvola Player 3.0.3. Thanks for your bug report.
